react/features/base/media/components/web/Video.js

```diff
@@ -203,14 +203,7 @@ class Video extends Component<Props> {
             // Ensure the video gets play() called on it. This may be necessary in the
             // case where the local video container was moved and re-attached, in which
             // case video does not autoplay.
-            this._videoElement.play()
-                .catch(error => {
-                    // Prevent uncaught "DOMException: The play() request was interrupted by a new load request"
-                    // when video playback takes long to start and it starts after the component was unmounted.
-                    if (this._mounted) {
-                        throw error;
-                    }
-                });
+            this._videoElement.play().then(null, noop);
```

Comment:
I think it's dangerous to just swallow the errors. How about filtering by browser type and maybe trying to match the specific error that is ignored instead?